extensions/amp-analytics/0.1/events.js

```diff
@@ -138,13 +138,22 @@ export class CustomEventTracker extends EventTracker {
         });
       }, 1);
     }
-
+    let selector = config['selector'];
+    if (!selector) {
+      selector = ':root';
+    }
+    const selectionMethod = config['selectionMethod'] || null;
+    const target = this.root.getElement(context, selector, selectionMethod);
```

Comment:
made the change. also added a note to `getElement` comment. I am thinking what about we making `getElement` handle the document-ready signal?